internals/logger/logger.go

```diff
@@ -58,24 +58,22 @@ func Panicf(format string, v ...any) {
 	loggerLock.Lock()
 	defer loggerLock.Unlock()
 	msg := fmt.Sprintf(format, v...)
-	logger.Notice("PANIC " + msg)
+	logger.Noticef("PANIC %s", msg)
```

Comment:
Can we not do what was done below with:

```go
l.Noticef("DEBUG "+format, v...)
```

To remove the Sprintf here too ?

Reply:
Yes but the panic below need the message, that's why I kept it, but after second thought, it's probably better to be consistent with the debug log. Updating now.